app/views/steps/completion/confirmation/show.en.html.erb

```diff
@@ -34,7 +40,7 @@
       </p>
 
       <% unless user_signed_in? || current_c100_application.receipt_email? %>
-        <div class="panel panel-border-narrow">
+        <div class="govuk-inset-text">
           <p>If you want to keep a copy, you must download it now. You cannot return to a submitted application.</p>
         </div>
       <% end %>
```

Comment:
The button to download the PDF is not migrated, use link_button helper (look for an example that also uses GA tracking as it needs the classes and data-module)